ChaosMod/stdafx.h

```diff
@@ -2,43 +2,45 @@
 

```

Comment:
Weird things going on here

Reply:
Believe this is just a merge conflict. I'll fix it rq